<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityBox</class>
 <widget class="QWidget" name="ActivityBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>

  <!-- Top search bar -->
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>7</y>
     <width>230</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
        border: 1px solid #444;
        border-radius: 10px;
        padding: 5px 10px;
        background-color: #222;
        color: #eee;
    }
    QLineEdit:focus {
        border: 1px solid #2574F5;
    }</string>
   </property>
   <property name="placeholderText">
    <string>Star ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>12</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: none;
border-image: url(&quot;:/SpaceKnappar/SearchButton(NotPressed).png&quot;) 0 0 0 0 stretch stretch;
}
QPushButton:pressed {
border-image: url(&quot;:/SpaceKnappar/SearchButton(NotPressed).png&quot;) 0 0 0 0 stretch stretch;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="favoriteButton">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>9</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0)</string>
   </property>
  </widget>

  <!-- Lists -->
  <widget class="QListWidget" name="favoriteList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>256</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0)</string>
   </property>
  </widget>
  <widget class="QListWidget" name="searchResultsList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0); color: #fff;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="usersList">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>60</y>
     <width>321</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0)</string>
   </property>
  </widget>

  <!-- Mass search -->
  <widget class="QLabel" name="massLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mass:                      -</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="massMin">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>min</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="massMax">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>max</string>
   </property>
  </widget>

  <!-- Type combobox -->
  <widget class="QComboBox" name="typeBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Type</string>
   </property>
   <item><property name="text"><string>O</string></property></item>
   <item><property name="text"><string>B</string></property></item>
   <item><property name="text"><string>A</string></property></item>
   <item><property name="text"><string>F</string></property></item>
   <item><property name="text"><string>G</string></property></item>
   <item><property name="text"><string>K</string></property></item>
   <item><property name="text"><string>M</string></property></item>
   <item><property name="text"><string>Supergiant</string></property></item>
   <item><property name="text"><string>Bright Giant</string></property></item>
   <item><property name="text"><string>Giant</string></property></item>
   <item><property name="text"><string>Subgiant</string></property></item>
   <item><property name="text"><string>Dwarf Star</string></property></item>
   <item><property name="text"><string>Subdwarf</string></property></item>
   <item><property name="text"><string>White Dwarf</string></property></item>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font><bold>true</bold></font>
   </property>
   <property name="text"><string>Type:</string></property>
  </widget>

  <!-- Menu buttons at bottom -->
  <widget class="QPushButton" name="searchMenuButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip"><string>Search</string></property>
   <property name="styleSheet"><string notr="true">background-color:rgba(0,0,0,0)</string></property>
  </widget>
  <widget class="QPushButton" name="helpMenuButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip"><string>Help</string></property>
   <property name="styleSheet"><string notr="true">background-color:rgba(0,0,0,0)</string></property>
  </widget>
  <widget class="QPushButton" name="favoriteMenuButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip"><string>Favourites</string></property>
   <property name="styleSheet"><string notr="true">background-color:rgba(0,0,0,0)</string></property>
  </widget>
  <widget class="QPushButton" name="usersMenuButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip"><string>Users</string></property>
   <property name="styleSheet"><string notr="true"/>
   </property>
  </widget>

  <!-- Labels -->
  <widget class="QLabel" name="favouritesLabel">
   <property name="geometry">
    <rect><x>20</x><y>10</y><width>251</width><height>31</height></rect>
   </property>
   <property name="font"><font><pointsize>20</pointsize></font></property>
   <property name="text"><string>Favourites</string></property>
   <property name="alignment"><set>Qt::AlignmentFlag::AlignCenter</set></property>
  </widget>
  <widget class="QLabel" name="Regisrationlable">
   <property name="geometry">
    <rect><x>40</x><y>10</y><width>251</width><height>31</height></rect>
   </property>
   <property name="font"><font><pointsize>20</pointsize></font></property>
   <property name="text"><string>Registered users</string></property>
   <property name="alignment"><set>Qt::AlignmentFlag::AlignCenter</set></property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect><x>70</x><y>310</y><width>58</width><height>16</height></rect>
   </property>
  </widget>

  <!-- Bottom/utility buttons -->
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry"><rect><x>280</x><y>10</y><width>61</width><height>31</height></rect></property>
   <property name="styleSheet"><string notr="true">QPushButton {
           border: none;
           border-image: url(:/SpaceKnappar/DeleteUser(NotPressed).png);
        }
        QPushButton:pressed {
           border-image: url(:/SpaceKnappar/DeleteUser(MidPress).png);
        }</string></property>
  </widget>
  <widget class="QPushButton" name="passwordButton">
   <property name="geometry"><rect><x>0</x><y>10</y><width>61</width><height>31</height></rect></property>
   <property name="styleSheet"><string notr="true">QPushButton {
           border: none;
           border-image: url(:/SpaceKnappar/ChangePassword(NotPressed).png);
        }
        QPushButton:pressed {
           border-image: url(:/SpaceKnappar/ChangePassword(MidPress).png);
        }</string></property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry"><rect><x>80</x><y>540</y><width>61</width><height>30</height></rect></property>
   <property name="toolTip"><string>Quit</string></property>
   <property name="styleSheet"><string notr="true">QPushButton {
           border: none;
           border-image: url(:/SpaceKnappar/Quit(NotPressed).png);
        }
        QPushButton:pressed {
           border-image: url(:/SpaceKnappar/Quit(Pressed).png);
        }</string></property>
  </widget>
  <widget class="QPushButton" name="BackToLogin">
   <property name="geometry"><rect><x>209</x><y>540</y><width>61</width><height>30</height></rect></property>
   <property name="toolTip"><string>Back to login</string></property>
   <property name="styleSheet"><string notr="true">QPushButton {
           border: none;
           border-image: url(:/SpaceKnappar/BackToLogin(NotPressed).png);
        }
        QPushButton:pressed {
           border-image: url(:/SpaceKnappar/BackToLogin(Pressed).png);
        }</string></property>
  </widget>
  <widget class="QPushButton" name="sunButton">
   <property name="geometry"><rect><x>140</x><y>590</y><width>61</width><height>30</height></rect></property>
   <property name="toolTip"><string>Back to sun</string></property>
   <property name="styleSheet"><string notr="true">QPushButton {
           border: none;
           border-image: url(:/SpaceKnappar/Sun(NotPressed).png);
        }
        QPushButton:pressed {
           border-image: url(:/SpaceKnappar/Sun(Pressed).png);
        }</string></property>
  </widget>
  <widget class="QPushButton" name="toggleCameraModeButton">
   <property name="geometry"><rect><x>60</x><y>410</y><width>211</width><height>24</height></rect></property>
   <property name="text"><string>Switch to First-Person View</string></property>
  </widget>

  <!-- Panels -->
  <widget class="QLabel" name="fillpanle">
   <property name="geometry"><rect><x>9</x><y>47</y><width>351</width><height>411</height></rect></property>
   <property name="styleSheet"><string notr="true">border: 2px solid #2574F5;
border-radius: 35px;
padding: 5px;
</string></property>
  </widget>
  <widget class="QLabel" name="bottomPanel">
   <property name="geometry"><rect><x>30</x><y>470</y><width>301</width><height>112</height></rect></property>
   <property name="styleSheet"><string notr="true">background-color:rgba(90,90,90,0.5);
border: 2px solid #2574F5;
border-radius: 35px;
padding: 5px;
</string></property>
  </widget>

  <!-- Help overlay content -->
  <widget class="QLabel" name="helpLabel">
   <property name="geometry"><rect><x>50</x><y>70</y><width>281</width><height>361</height></rect></property>
  </widget>
  <widget class="QLabel" name="Help1_label">
   <property name="geometry"><rect><x>50</x><y>60</y><width>181</width><height>31</height></rect></property>
   <property name="font"><font><pointsize>12</pointsize><bold>true</bold></font></property>
   <property name="text"><string>Hjälp Meny:</string></property>
  </widget>
  <widget class="QLabel" name="Help2_label">
   <property name="geometry"><rect><x>150</x><y>110</y><width>201</width><height>31</height></rect></property>
   <property name="text"><string>Styra kameran (Låsläge)</string></property>
  </widget>
  <widget class="QLabel" name="Help6_label">
   <property name="geometry"><rect><x>150</x><y>180</y><width>191</width><height>20</height></rect></property>
   <property name="text"><string>Styra kameran (Friläge)</string></property>
  </widget>
  <widget class="QLabel" name="Help5_label">
   <property name="geometry"><rect><x>150</x><y>230</y><width>211</width><height>31</height></rect></property>
   <property name="text"><string>Hovra: Sjärnornas ID</string></property>
  </widget>
  <widget class="QLabel" name="Help3_label">
   <property name="geometry"><rect><x>150</x><y>290</y><width>201</width><height>31</height></rect></property>
   <property name="text"><string>Enkel klick: Info om sjärnorna</string></property>
  </widget>
  <widget class="QLabel" name="Help4_label">
   <property name="geometry"><rect><x>150</x><y>350</y><width>201</width><height>31</height></rect></property>
   <property name="text"><string>Dubbel klick: Åka till sjärnorna</string></property>
  </widget>

  <!-- Help icons -->
  <widget class="QLabel" name="Icon_1">
   <property name="geometry"><rect><x>50</x><y>110</y><width>49</width><height>41</height></rect></property>
   <property name="styleSheet"><string notr="true">border: none;
border-image: url(:/Help_knappar/Keybinds.png);</string></property>
  </widget>
  <widget class="QLabel" name="Icon_4">
   <property name="geometry"><rect><x>50</x><y>170</y><width>49</width><height>41</height></rect></property>
   <property name="styleSheet"><string notr="true">border: none;
border-image: url(:/Help_knappar/kamera_move.png);</string></property>
  </widget>
  <widget class="QLabel" name="Icon_5">
   <property name="geometry"><rect><x>50</x><y>230</y><width>49</width><height>41</height></rect></property>
   <property name="styleSheet"><string notr="true">border: none;
border-image: url(:/Help_knappar/Hover.png);</string></property>
  </widget>
  <widget class="QLabel" name="Icon_2">
   <property name="geometry"><rect><x>50</x><y>290</y><width>49</width><height>41</height></rect></property>
   <property name="styleSheet"><string notr="true">border: none;
border-image: url(:/Help_knappar/Enkel.png);</string></property>
  </widget>
  <widget class="QLabel" name="Icon_3">
   <property name="geometry"><rect><x>50</x><y>350</y><width>49</width><height>41</height></rect></property>
   <property name="styleSheet"><string notr="true">border: none;
border-image: url(:/Help_knappar/Dubbel.png);</string></property>
  </widget>

  <!-- Z‑order (render order) -->
  <zorder>helpLabel</zorder>
  <zorder>fillpanle</zorder>
  <zorder>bottomPanel</zorder>
  <zorder>searchLineEdit</zorder>
  <zorder>searchButton</zorder>
  <zorder>favoriteButton</zorder>
  <zorder>favoriteList</zorder>
  <zorder>massLabel</zorder>
  <zorder>typeBox</zorder>
  <zorder>typeLabel</zorder>
  <zorder>massMin</zorder>
  <zorder>massMax</zorder>
  <zorder>searchResultsList</zorder>
  <zorder>searchMenuButton</zorder>
  <zorder>favouritesLabel</zorder>
  <zorder>helpMenuButton</zorder>
  <zorder>usersMenuButton</zorder>
  <zorder>usernameLabel</zorder>
  <zorder>usersList</zorder>
  <zorder>Regisrationlable</zorder>
  <zorder>deleteButton</zorder>
  <zorder>quitButton</zorder>
  <zorder>BackToLogin</zorder>
  <zorder>favoriteMenuButton</zorder>
  <zorder>passwordButton</zorder>
  <zorder>sunButton</zorder>
  <zorder>toggleCameraModeButton</zorder>
  <zorder>Help1_label</zorder>
  <zorder>Help2_label</zorder>
  <zorder>Help6_label</zorder>
  <zorder>Help5_label</zorder>
  <zorder>Help3_label</zorder>
  <zorder>Help4_label</zorder>
  <zorder>Icon_1</zorder>
  <zorder>Icon_4</zorder>
  <zorder>Icon_5</zorder>
  <zorder>Icon_2</zorder>
  <zorder>Icon_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
